packages/react/src/components/TextInput/TextInput.js

```diff
@@ -94,8 +104,10 @@ const TextInput = React.forwardRef(function TextInput(
           <WarningFilled16 className={`${prefix}--text-input__invalid-icon`} />
         )}
         {input}
+        <hr className={`${prefix}--text-input__divider`} />
```

Comment:
Does the divider need to be conditional now based on `isFluid` or is this also an addition to the base text input as well?

Reply:
No it has to be conditional. Only in isFluid.